src/apps/backend/modules/account/rest-api/account-router.ts

```diff
@@ -1,5 +1,5 @@
-import { accessAuthMiddleware } from '../../access-token/rest-api/access-auth-middleware';
 import { ApplicationRouter } from '../../application';
+import { accessAuthMiddleware } from '../../authentication/access-token/rest-api/access-auth-middleware';
```

Comment:
Anything consumed by other modules should be public and at the root folder of the module. In this case, `access-auth-middleware` should be at the root level of authentication module